From 61579d05902345fe37e26a648620d719f22c6226 Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Wed, 16 Dec 2020 16:15:20 +0530
Subject: [PATCH 0980/1921] octeontx2-pf: Update RVU PF/VF driver name

This patch removes platform specific name from the
PF/VF driver name to make it appropriate for all
supported platforms

Change-Id: I5e7ec07ae9bfe7a1a1be9ad4f21912ceca498475
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42414
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_pf.c | 4 ++--
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_vf.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_pf.c b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_pf.c
index 3cbd43a10a85..2676cea71e0c 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_pf.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_pf.c
@@ -27,8 +27,8 @@
 #include "cn10k.h"
 #include <rvu_trace.h>
 
-#define DRV_NAME	"octeontx2-nicpf"
-#define DRV_STRING	"Marvell OcteonTX2 NIC Physical Function Driver"
+#define DRV_NAME	"RVU-nicpf"
+#define DRV_STRING	"Marvell RVU NIC Physical Function Driver"
 #define DRV_VERSION	"1.0"
 
 /* Supported devices */
diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_vf.c b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_vf.c
index b4b618277096..559da2feff22 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_vf.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_vf.c
@@ -16,8 +16,8 @@
 #include "otx2_reg.h"
 #include "cn10k.h"
 
-#define DRV_NAME	"octeontx2-nicvf"
-#define DRV_STRING	"Marvell OcteonTX2 NIC Virtual Function Driver"
+#define DRV_NAME	"RVU-nicvf"
+#define DRV_STRING	"Marvell RVU NIC Virtual Function Driver"
 #define DRV_VERSION	"1.0"
 
 static const struct pci_device_id otx2_vf_id_table[] = {
-- 
2.31.1

